GoMeet/src/main/webapp/js/fetch-wrapper.js

```diff
@@ -7,4 +7,9 @@ class FetchWrapper {
     let response = await fetch(servlet, {method: 'POST', body: params});
     return response;
   }
+
+  async doGet(servlet) {
```

Comment:
It might be a good idea to add some comments to specify what the param `servlet` is (i.e. the URL that the target servlet is mapped to).

Reply:
FetchWrapper.js has been removed from PR #29 onwards but, in future, I will add JSDoc for any parameters that might not be clear :)